prime-router/src/main/kotlin/fhirengine/translation/HL7toFhirTranslator.kt

```diff
@@ -122,11 +130,17 @@ class HL7toFhirTranslator internal constructor(
                 bundle.addEntry().resource = Patient()
                 bundle.entry.first { it.resource.resourceType.name == "Patient" }.resource as Patient
             }
+
+            if ((birthTime?.length ?: 0) <= 8) {
+                // this field is just a date with no time, no need to create an extension to track the time
+                return
```

Comment:
The date needs to be added in here since the mapping for the date was removed.

Reply:
This just handles the birth time extension.  The base `birthDate` still works through the `Patient.yml` file.  That part was not removed.

https://github.com/CDCgov/prime-reportstream/blob/35366ec302a4df4d579ceab5bc8922dff1c36ada/prime-router/metadata/fhir_mapping/hl7/resource/Patient.yml#L260-L263

Reply:
Yep, you are right, sorry about that.